### DISCUSSION TITLE
Material components

### DISCUSSION THREAD

**Discussion by neoacevedo** [schultek/jaspr#328] [Q&A]
How may I to use some material components? E.g,  in `app.dart` below the main div I add the following:

```dart
 yield script(src: '', attributes: {
      'type': 'importmap'
    }, [
      RawText(
          '{"imports": {"@material/web/": "https://esm.run/@material/web/"}}')
    ]);

    yield script(src: '', attributes: {
      'type': 'importmap'
    }, [
      RawText('import \'@material/web/all.js\';'),
      RawText(
          'import {styles as typescaleStyles} from \'@material/web/typography/md-typescale-styles.js\'')
    ]);
```

This should allow me to use material components, but, how should I to do to use them?

**Comment by rodydavis** [comment]
I used to do Material Web DevRel, and the best bet would be to include the packages in an import script and then use the web components directly in the HTML.

You can create them with a custom tag.
